lua/gk.lua

```diff
@@ -30,6 +30,12 @@ return function (net_conf, lls_conf, sol_conf, gk_lcores)
 	local gk_max_pkt_burst_front = 32
 	local gk_max_pkt_burst_back = 32
 
+	-- The rate and burst size of the icmp messages.
+	local front_icmp_msgs_per_sec = 1000
+	local front_icmp_msgs_burst = 50
```

Comment:
Are these values the defaults of the Linux kernel?

Reply:
Yes, they are. The kernel definitions are ``int sysctl_icmp_msgs_per_sec __read_mostly = 1000;`` and ``int sysctl_icmp_msgs_burst __read_mostly = 50;``